### PR TITLE
Fix typo `I18n::translate` method name

### PR DESCRIPTION
**Pull request by afbora**
### Fixes
- Fix typo `I18n::translate` method name #5193

### Breaking changes
None

## Ready?

- [ ] Unit tests for fixed bug/feature
- [ ] In-code documentation (wherever needed)
- [ ] Tests and checks all pass

### For review team

- [ ] Add changes to release notes draft in Notion
- [ ] Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)

### CODE REVIEW

**Comment by afbora** [comment]
Also includes minor composer.json update

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "getkirby/cms",
-	"description": "The Kirby 3 core",
+	"description": "The Kirby 4 core",
```
Comment (lukasbestle) [inline composer.json]: I'd vote to leave out the version number entirely. Then we can't forget it. 😅

Comment (afbora) [reply composer.json]: I think it's a solution that will forever 💯

**Review by distantnative — DISMISSED** [review]

**Comment by distantnative** [comment]
Oh wait, shouldn't this be fixed on `v4/develop` rather?

**Comment by afbora** [comment]
Moved https://github.com/getkirby/kirby/pull/5196

**Pull request closed without merging**